### PR TITLE
Showing video from AME #1

### PR DESCRIPTION
**Pull request by McRo**
Card #1 - Branch Feature/showing ame video
Artwork page
Update template stucture for Bootstrap 3
Link with Ame api serveur
Playing Video from Ame serveur
Disable archive icon when server error (not correct ip)
/!\ for testing need to have Fresnoy IP
/!\ need new config.js

### CODE REVIEW

File: app/scripts/controllers.coffee [hunk 1]
```diff
@@ -97,24 +97,22 @@ angular.module('memoire.controllers', ['memoire.services'])
...
             picture : archive.flvthumb
             medium_url : $sce.trustAsResourceUrl(archive.flvpath)
-            description : ''
+            description : archive.field8
          })
-         console.log($scope.ame_artwork_gallery)
+
     , (response) ->
-      console.log(response)
+      #erreur service
```
Comment (glibersat) [inline app/scripts/controllers.coffee]: clean

File: bower.json [hunk 2]
```diff
@@ -21,6 +21,7 @@
     "tests"
   ],
   "dependencies": {
+    "slug" : "",
```
Comment (glibersat) [inline bower.json]: version?

File: app/scripts/controllers.coffee [hunk 3]
```diff
@@ -97,24 +97,22 @@ angular.module('memoire.controllers', ['memoire.services'])
         )
 
 
-    search = artwork.title
+    search = (artwork.title)+" "+slug(artwork.authors[0].user.first_name[0]+artwork.authors[0].user.last_name).toLowerCase()
+    #search :  Archipel gabbruzzese
```
Comment (glibersat) [inline app/scripts/controllers.coffee]: ?

File: app/scripts/controllers.coffee [hunk 4]
```diff
@@ -97,24 +97,22 @@ angular.module('memoire.controllers', ['memoire.services'])
         )
 
 
-    search = artwork.title
+    search = (artwork.title)+" "+slug(artwork.authors[0].user.first_name[0]+artwork.authors[0].user.last_name).toLowerCase()
```
Comment (glibersat) [inline app/scripts/controllers.coffee]: why an empty string?

**Comment by glibersat** [comment]
fix the conflicts first

File: app/scripts/controllers.coffee (hunk 3 above)
Comment (McRo) [inline app/scripts/controllers.coffee]: This is an example of search

File: app/scripts/controllers.coffee (hunk 4 above)
Comment (McRo) [inline app/scripts/controllers.coffee]: Ame api have no option "exact" search
For example when i seach "Archipel" i have 2 Aw from Giacomo Abbruzzese and one from Daphné Hérétakis named "Archipels, granites dénudés"
That why i add and author path name (ame api search on every field)

**Pull request closed without merging**